ktor-client/ktor-client-core/common/src/io/ktor/client/features/websocket/WebSockets.kt

```diff
@@ -5,29 +5,104 @@
 package io.ktor.client.features.websocket
 
 import io.ktor.client.*
+import io.ktor.client.call.*
+import io.ktor.client.engine.*
 import io.ktor.client.features.*
 import io.ktor.client.request.*
 import io.ktor.client.statement.*
 import io.ktor.http.*
 import io.ktor.http.cio.websocket.*
 import io.ktor.util.*
+import kotlin.native.concurrent.*
+
+@ExperimentalWebSocketExtensionApi
+@SharedImmutable
+private val REQUEST_EXTENSIONS_KEY = AttributeKey<List<WebSocketExtension<*>>>("Websocket extensions")
+
+/**
+ * Indicates if a client engine supports WebSockets.
+ */
+public object WebSocketCapability : HttpClientEngineCapability<Unit> {
+    override fun toString(): String = "WebSocketCapability"
```

Comment:
I wonder why don't we do the same as for attribute key? I mean, why don't we pass this constant through the constructor instead?
Just question, not necessary to fix

Reply:
Let's fix separately the AttributeKey. It's `[object Object]` in `kotlin-js`